### DISCUSSION TITLE
Some quality of life ideas

### DISCUSSION THREAD

**Discussion by theblacktiger59** [comfyanonymous/ComfyUI#147] [Ideas]
I wanna start by saying thanks for this great tool! I understand the diffusion process better and enjoy the tool's control.
I have a few quality of life type of ideas:

- Having a toggle button SAVE on the save picture node, I use a 3 passes workflow and want to see the intermediate steps but not always save them.
- An option to save image as JPEG
- Choose output folders for image save
- Have the node movement / resize snap to a grid.
- input to float / int value of the different nodes, thinking of the size X and Y of the EmptyLantentImage and LatentUpscale, that plus a simple float/int node with an output and a math node would allow changing the resolution (both low and upscaled) with only one change of value. Could be useful to also tight together different cgf, denoise value of the samplers of different steps (multi-passes workflow)

PS: Added a screen of my current workflow to illustrate how I could make use of those ideas perhaps.

![image](https://user-images.githubusercontent.com/35955523/226096054-5d1555b2-13f7-4b5b-a092-dab603c12d3e.png)

**Comment by WASasquatch** [comment]
> I wanna start by saying thanks for this great tool! I understand the diffusion process better and enjoy the tool's control. I have a few quality of life type of ideas:
> 
>     * Having a toggle button SAVE on the save picture node, I use a 3 passes workflow and want to see the intermediate steps but not always save them.
> 
>     * An option to save image as JPEG
> 
>     * Choose output folders for image save
> 
>     * Have the node movement / resize snap to a grid.
> 
>     * input to float / int value of the different nodes, thinking of the size X and Y of the EmptyLantentImage and LatentUpscale, that plus a simple float/int node with an output and a math node would allow changing the resolution (both low and upscaled) with only one change of value. Could be useful to also tight together different cgf, denoise value of the samplers of different steps (multi-passes workflow)
> 
> 
> PS: Added a screen of my current workflow to illustrate how I could make use of those ideas perhaps.
> 
> ![image](https://user-images.githubusercontent.com/35955523/226096054-5d1555b2-13f7-4b5b-a092-dab603c12d3e.png)

Having a save button sorta breaks the automated nature of the "SaveImage" node, though I have requested a Base64 based "View Image" node which would just show the image inline, without saving it. That would stick to the core ideas of a node system and execution. 

Rest of these ideas are good.

**Reply by omar92** [reply]
the save option can be boolean
